### PR TITLE
[front] - refacto(env): DUST_CLIENT_FACING_URL as next

### PR DESCRIPTION
**Pull request by JulesBelveze**
## Description

Use `NEXT_PUBLIC_DUST_CLIENT_FACING_URL` directly from front to avoid drilling down `dustClientFacingUrl` variable.

## Risk

None, if rollout is properly done

## Deploy Plan

- [x] Set `DUST_CLIENT_FACING_URL` in front k8s secrets
- [ ] Apply `infra`
- [ ] Deploy & test `front-edge`
- [ ] Deploy `front`
- [ ] Update dev setup

### CODE REVIEW

File: k8s/cloudbuild.yaml [hunk 1]
```diff
@@ -17,6 +17,8 @@ steps:
       - NEXT_PUBLIC_VIZ_URL=https://viz.dust.tt
       - --build-arg
       - NEXT_PUBLIC_GA_TRACKING_ID=G-K9HQ2LE04G
+      - --build-arg
+      - NEXT_PUBLIC_DUST_CLIENT_FACING_URL=https://dust.tt
```
Comment (PopDaph) [inline k8s/cloudbuild.yaml:21]: This used for all env so probably not good, let's check with @fontanierh

**Review by PopDaph — APPROVED** [review]
LGTM but let's also wait from validation from our infra guy! 
You just need to update the deploy plan (need to apply infra changes)

**Review by fontanierh — APPROVED** [review]
Looks good !

**Comment by PopDaph** [comment]
@JulesBelveze can you link the follow up PR? So that if someone takes this as example she can fall into the pit of success and do it one shot 😄

**Comment by JulesBelveze** [comment]
The follow up *PRs* you mean haha.
Here you go:
- https://github.com/dust-tt/dust/pull/7502
- https://github.com/dust-tt/dust/pull/7503
- https://github.com/dust-tt/dust/pull/7505

**Pull request merged**